sensu-report

```diff
@@ -76,6 +76,33 @@ def fetch_sensu_data(server, port, client, username, password):
     data = json.load(response)
     return data
 
+def check_client_exists(server, port, client, username, password):
```

Comment:
Can you make this just return a bool? I think it would be easier to understand and interact with instead of making the caller "remember" that this function returns an http code.